<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>424</width>
    <height>504</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="butVm">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>401</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ver media de todos los puntos extras obtenidos </string>
   </property>
  </widget>
  <widget class="QPushButton" name="butonSalir">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>280</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Salir</string>
   </property>
  </widget>
  <widget class="QPushButton" name="butVc">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>401</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ver cantidad de estudiantes registrados</string>
   </property>
  </widget>
  <widget class="QPushButton" name="butVd">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>80</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ver Datos de un estudiante </string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>281</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Bienvenido de nuevo</string>
   </property>
  </widget>
  <widget class="QLabel" name="revisar">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>330</y>
     <width>301</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="butGraf1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>401</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ver gráfico de los estudiantes con más exámenes realizados</string>
   </property>
  </widget>
  <widget class="QPushButton" name="butGraf2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>401</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ver gráfico de los estudiantes con más puntos extras obtenidos</string>
   </property>
  </widget>
  <widget class="QLabel" name="revisar_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>380</y>
     <width>301</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
